src/logger.js

```diff
@@ -1,25 +1,24 @@
 import winston from 'winston';
-import { getEnv } from './config';
 
-const loggerConsoleOptions = {
-  timestamp: false,
-  level: getEnv('LOG_LEVEL'),
-  colorize: false,
-  formatter: options => `${options.message}`,
+const logger = (logLevel) => {
```

Comment:
Why do you think importing config variables is bad, and make every thing parameter is better?